panic: runtime error: slice bounds out of range [:-1]

Hi pretty new to github so tell me if im missing things. i installed ffuf on windows but everytime i try to run it with a url and wordlist file i get this error.

C:\Users\xxx>ffuf -u google.com/FUZZ/ -w test.com
panic: runtime error: slice bounds out of range [:-1]

goroutine 1 [running]:
main.prepareConfig(0xc0000cc000, 0xc000058d00, 0x4, 0x7)
        /home/joona/go/src/github.com/ffuf/ffuf/main.go:314 +0x26f2
main.main()
        /home/joona/go/src/github.com/ffuf/ffuf/main.go:143 +0x19fb

I googled it but cant come up with a solution
Thank you

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -14,6 +14,7 @@
     - Fixed the issue where the option -ac was overwriting existing filters. Now auto-calibration will add them where needed.
     - The `-w` flag now accepts comma delimited values in the form of `file1:W1,file2:W2`.
     - Links in the HTML report are now clickable
+    - Fixed panic during wordlist flag parsing in Windows systems.
 
 - v1.1.0
   - New

--- a/pkg/ffuf/optionsparser.go
+++ b/pkg/ffuf/optionsparser.go
@@ -176,7 +176,11 @@ func ConfigFromOptions(parseOpts *ConfigOptions, ctx context.Context, cancel con
 				// The wordlist was supplied without a keyword parameter
 				wl = []string{v}
 			} else {
-				filepart := v[:strings.LastIndex(v, ":")]
+				filepart := v
+				if strings.Contains(filepart, ":") {
+					filepart = v[:strings.LastIndex(filepart, ":")]
+				}
+
 				if FileExists(filepart) {
 					wl = []string{filepart, v[strings.LastIndex(v, ":")+1:]}
 				} else {

--- a/pkg/ffuf/util.go
+++ b/pkg/ffuf/util.go
@@ -31,11 +31,13 @@ func UniqStringSlice(inslice []string) []string {
 	return ret
 }
 
-//FileExists checks if the filepath exists and is not a directory
+//FileExists checks if the filepath exists and is not a directory.
+//Returns false in case it's not possible to describe the named file.
 func FileExists(path string) bool {
 	md, err := os.Stat(path)
-	if os.IsNotExist(err) {
+	if err != nil {
 		return false
 	}
+
 	return !md.IsDir()
 }

